### ISSUE TITLE
Repeated keyword arguments allowed when function has spread keyword parameters

### ISSUE THREAD

**Issue by layday** [microsoft/pyright#1440]
**Describe the bug**
When a function defines `**kwargs`, keyword arguments can be repeated in function calls without producing an error.

**To Reproduce**
Type check the snippet below.

**Expected behavior**
That `bar(val=None, val=None)` would produce a similar error to `foo(val=None, val=None)`.

**Screenshots or Code**
```python
def foo(val: object): ...

def bar(**kwargs: object): ...

foo(val=None, val=None)  # 'Parameter "val" is already assigned'
bar(val=None, val=None)  # No error from Pyright, still a SyntaxError
```

**VS Code extension or command-line**
Either, 1.1.106

**Comment by erictraut** [comment]
This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.107, which I just published. It will also be included in the next release of Pylance.
